website/content/api-docs/system/internal-counters.mdx

```diff
@@ -875,6 +877,15 @@ The `/sys/internal/counters/config` endpoint is used to configure logging of act
 
 Any missing parameters are left at their existing value.
 
+<Note>
+
+When [automated utilization
+reporting](/vault/docs/enterprise/license/utilization-reporting) is enabled, the
+`retention_months` parameter cannot be configured below the default setting (24)
```

Comment:
I think we'll want this to reflect the `minimum_retention_months` setting as well, rather than "24". Maybe something like:

```suggestion
`retention_months` parameter cannot be configured below the default setting ([minimum_retention_months](#read-the-client-count-configuration)).
```

I'm not sure if we want a direct reference here, or if the anchoring is even correct 😅, so I defer to you on the actual change!